File: MarkerOnGooleMaps/gsg/ViewController.m
//
//  ViewController.m
//  gsg
//
//  Created by yosemite on 22/01/15.
//  Copyright (c) 2015 yosemite. All rights reserved.
//

#import "ViewController.h"
#import <GoogleMaps/GoogleMaps.h>

@interface ViewController ()

@end

@implementation ViewController
{
    //INSTACIAR EL MAPAVIEW
    GMSMapView *mapView_;
}

- (void)viewDidLoad {
    [super viewDidLoad];

    // Do any additional setup after loading the view, typically from a nib.
//CREAR MAPA con posicion
    //mark llamar la clase que obtiene la posicion del dispositivo
    GMSCameraPosition *camera =[GMSCameraPosition cameraWithLatitude:18.39 longitude:-93.20 zoom:6];
    mapView_ = [GMSMapView mapWithFrame:CGRectZero camera:camera];
    mapView_.myLocationEnabled=YES;
    self.view = mapView_;
    
    
//CREAR UN MARCADOR EN EL MAPA
    GMSMarker *maker = [[GMSMarker alloc] init];
    maker.position = CLLocationCoordinate2DMake(18.39, -93.20);
    maker.title = @"Chedraui Paraiso";
    maker.snippet=@"Tabasco";
    maker.map = mapView_;
    
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Swift_MarkerOnGoogleMaps/gsg_s_m1/ViewController.swift
//
//  ViewController.swift
//  gsg_s_m1
//
//  Created by yosemite on 22/01/15.
//  Copyright (c) 2015 yosemite. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    //INSTA
   
    override func viewDidLoad() {
        super.viewDidLoad()
      
        let latitude1 = 17.0
        let longitude1 = 96.16
        //Instancias el mapa
        var camera = GMSCameraPosition.cameraWithLatitude(latitude1, longitude: longitude1, zoom: 8)
        var mapView = GMSMapView.mapWithFrame(CGRectZero, camera: camera)
        mapView.myLocationEnabled=true
        self.view=mapView;
        
        //Crear un marcador en el mapa
        var marker = GMSMarker()
        marker.position=CLLocationCoordinate2DMake(latitude1,longitude1)
        marker.title="GSG"
        marker.snippet="Paraiso";
        marker.map = mapView
        
        
        
           }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    //METODOS DELEGADOS PARA LOCALIZACION



}

